data/macos.yaml

```diff
@@ -380,7 +416,7 @@ name: MacOSMailBackupTOC
 doc: Mail BackupTOC
```

Comment:
I know this is not part of your changes but could you update this to "Mail Backup Table of Contents (TOC)", thx in advance.

Reply:
fixed